trello_client.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime
+import requests
+import os
+
+from item import Item
+from trello_data import TrelloData
+from trello_ids import TrelloIDs
+
+class TrelloClient:
+
+    def __init__(self):
+
+        self.board_id = os.environ.get('BOARD_ID')        
+        self.auth_params_key = os.environ.get('AUTH_PARAMS_KEY')
+        self.auth_params_token = os.environ.get('AUTH_PARAMS_TOKEN')
+        self.auth_params = {'key': self.auth_params_key, 'token': self.auth_params_token}
+        
+        self.idList_todo = ''
+        self.idList_doing = ''
+        self.idList_done = ''
+
+        self.cards = []
+        self.lists = []
+        self.items = []
+
+        self.refresh_items()
+        self.refresh_ids()
+
+
+    def refresh_items(self):
+        
+        trello_data = TrelloData(self.board_id, self.auth_params)
+        self.cards = trello_data.cards
+        self.lists = trello_data.lists
+
+        # Retrieve relevant info on all to do items
+        self.items = []
+        for card in self.cards:
+            for list in self.lists:
+                if card['idList'] == list['id']:
+                    # The 'status' of the card is the name of the list it's in.
+                    status = list['name']
+                    break
+            last_modified = datetime.strptime(card['dateLastActivity'][:-1] + '000', '%Y-%m-%dT%H:%M:%S.%f')
+            item = Item(card['id'], card['name'], status, last_modified)
+            self.items.append(item)
+
+    def refresh_ids(self):
```

Comment:
Minor, but I'd be tempted to call this `load_ids` if its only expected to be called once at startup, to make the intention a little clearer